packages/expo-battery/android/src/main/java/expo/modules/battery/BatteryModule.kt

```diff
@@ -0,0 +1,108 @@
+package expo.modules.battery
+
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.BatteryManager
+import android.os.Build
+import android.os.PowerManager
+import org.unimodules.core.ExportedModule
+import org.unimodules.core.ModuleRegistry
+import org.unimodules.core.Promise
+import org.unimodules.core.interfaces.ExpoMethod
+import org.unimodules.core.interfaces.RegistryLifecycleListener
+import org.unimodules.core.interfaces.services.EventEmitter
+
+class BatteryModule(context: Context) : ExportedModule(context), RegistryLifecycleListener {
+  private val NAME = "ExpoBattery"
+  private var moduleRegistry: ModuleRegistry? = null
```

Comment:
```suggestion
```
We don't use this property outside of `onCreate` function. So it can be removed 😉